Add Project Loved articles (#2205)
ppy/osu-wiki: wiki/People/Project_Loved_Team/en.md

# Project Loved Team

> The **Project Loved Team** are members of the osu! community who manage [Project Loved](/wiki/Project_Loved), the voting platform that promotes beatmaps to the [Loved](/wiki/Beatmaps#loved) category.

Reviewer: ```suggestion
The **Project Loved Team** are members of the osu! community who manage [Project Loved](/wiki/Project_Loved), the voting project that promotes beatmaps to the [Loved](/wiki/Beatmaps#loved) category.
```
dont know if necessary

Author: think "platform" makes more sense (the "project" part has a wider scope than voting but the voting is all that's important for public)

## Roles and Responsibilities

> Most of the Project Loved Team consists of **captains** for each of the [Game Modes](/wiki/Game_Modes). Captains are responsible for nominating beatmaps for the Loved category. They also write comments about the nominated maps to help readers get a feel for each map in case they haven't seen or played them before.

Reviewer 2: ```suggestion
Most of the Project Loved Team consists of **captains** for each of the [Game Modes](/wiki/Game_Modes). Captains are responsible for nominating beatmaps for the Loved category. They also write comments about the nominated maps to help readers get a feel for each map in case they have not seen or played them before.
```

> Most of the Project Loved Team consists of **captains** for each of the [Game Modes](/wiki/Game_Modes). Captains are responsible for nominating beatmaps for the Loved category. They also write comments about the nominated maps to help readers get a feel for each map in case they have not seen or played them before.

Reviewer: ```suggestion
Project Loved Team mostly consists of **captains** for each [Game Mode](/wiki/Game_Modes). Captains are responsible for nominating beatmaps to the Loved category. They also write short descriptions about the nominated maps to help readers get a feel for each map in case they have not seen or played them before.
```

Author: can u add "The " at the front

The rest of the team helps with quality control, maintenance, and organisation.

## Team Members

| Name | Role |
| :-- | :-- |
| ![][flag_CL] [-Kazu-](https://osu.ppy.sh/users/920861) | osu!taiko captain |
| ![][flag_GB] [-Roxas](https://osu.ppy.sh/users/1986262) | osu! captain |
| ![][flag_US] [BeasttrollMC](https://osu.ppy.sh/users/3171691) | osu! captain |
| ![][flag_GB] [Bubbleman](https://osu.ppy.sh/users/5182050) | osu! captain |
| ![][flag_US] [clayton](https://osu.ppy.sh/users/3666350) | News and forum post writer, Mapper contact |
| ![][flag_US] [DigitalHypno](https://osu.ppy.sh/users/4384207) | osu! captain |

> | ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335) | Beatmap updater (when others can't!) |

Reviewer 2: ```suggestion
| ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335) | Beatmap updater (when others cannot!) |
```

> | ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335) | Beatmap updater (when others cannot!) |

Reviewer: ```suggestion
| ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335) | Beatmap updating |
```

Author: using nouns instead of verbs for these roles but I will remove that extra note

# Project Loved Team

The **Project Loved Team** are members of the osu! community who manage [Project Loved](/wiki/Project_Loved), the voting platform that promotes beatmaps to the [Loved](/wiki/Beatmaps#loved) category.

## Roles and Responsibilities

The Project Loved Team mostly consists of **captains** for each [game mode](/wiki/Game_Modes). Captains are responsible for nominating beatmaps to the Loved category. They also write short descriptions about the nominated maps to help readers get a feel for each map in case they have not seen or played them before.

The rest of the team helps with quality control, maintenance, and organisation.

> ## Team Members

Reviewer 3: I think this can be divided the same way as pinned on the discord server, per game mode and a part with the rest of the helpers

Author: the reason I didn't do this was because I thought it looked weird to have lists for the captains and then a table for the rest (because they have name + role)... should I just do that anyway, idk if it's actually a big concern lol

Reviewer 3: Eh, we can go with this, was just a comment :p

| Name | Role |
| :-- | :-- |
| ![][flag_CL] [-Kazu-](https://osu.ppy.sh/users/920861) | osu!taiko captain |
| ![][flag_GB] [-Roxas](https://osu.ppy.sh/users/1986262) | osu! captain |
| ![][flag_US] [BeasttrollMC](https://osu.ppy.sh/users/3171691) | osu! captain |
| ![][flag_GB] [Bubbleman](https://osu.ppy.sh/users/5182050) | osu! captain |

> | ![][flag_US] [clayton](https://osu.ppy.sh/users/3666350) | News and forum post writer, Mapper contact |

Reviewer: ```suggestion
| ![][flag_US] [clayton](https://osu.ppy.sh/users/3666350) | News and forum post writer, mapper contact |
```
